extensions/amp-list/0.1/test/test-amp-list.js

```diff
@@ -1153,6 +1153,43 @@ describes.repeated(
             });
           });
 
+          describe('binding="refresh-evaluate"', () => {
+            beforeEach(() => {
+              element.setAttribute('binding', 'refresh-evaluate');
+            });
+
+            it('should rescan() with {update: false, evaluate: true} before FIRST_MUTATE', async () => {
```

Comment:
nix the `update:false`. Maybe you meant `fast: true`?

Reply:
Oops, thanks.